app/forms/validation/mappings/Formatters.scala

```diff
@@ -40,37 +40,45 @@ trait Formatters {
   private[mappings] def currencyFormatter(requiredKey: String,
                                           invalidNumericKey: String,
                                           maxAmountKey: String,
+                                          minAmountKey: Option[String] = None,
```

Comment:
You don't need a default value there as you always set it when you call this method.